usaspending_api/search/delta_models/award_search.py

```diff
@@ -226,7 +226,7 @@
 
   TREASURY_ACCT.tas_paths,
   TREASURY_ACCT.tas_components,
-  SORT_ARRAY(TREASURY_ACCT.disaster_emergency_fund_codes),
+  TREASURY_ACCT.disaster_emergency_fund_codes,
```

Comment:
Does removing the sort help performance, or fix some kind of data issue? 
If it is helping in that way, does it hurt in terms of comparison fidelity -- causing perceived data discrepancies in unit tests or manual testing?

Reply:
This sort was moved into the JOIN to make sure we are getting the same data between Postgres and Spark. This probably should be refactored in the future, but to make data comparison easier I added this:
```
IF (
        SIZE(COLLECT_SET(faba.disaster_emergency_fund_code)) > 0,
        SORT_ARRAY(COLLECT_SET(faba.disaster_emergency_fund_code)),
        NULL
    ) AS disaster_emergency_fund_codes,
```
Overall it doesn't appear to affect performance.